phpstan.neon.dist

```diff
@@ -1,23 +1,13 @@
+includes:
+    - larastan.neon
```

Comment:
I'm wondering, can't we just reference the config _shipped_ with larastan that way? 🤷‍♀️

Reply:
Feel free to experiment with it. Actually I promote phpstan, I'm not a graphql user.

Reply:
`vendor/nunomaduro/larastan/extension.neon`

Reply:
I do not know how to _un-ignore_ errors.

Reply:
> I do not know how to _un-ignore_ errors.

Ahh. I guess that what the reason in the first place why I assimilated it.